app/helpers/configuration_helper/configuration_view_helper.rb

```diff
@@ -88,5 +88,9 @@ def list_view(resource)
        inactive_icon("view_tile.png", _('Tile View'), resource, "tile"),
        active_icon("view_list.png", _('List View'))].compact.join('')
     end
+
+    def check_roles(arr)
+      arr.detect { |r| role_allows(:feature => r) }
```

Comment:
It doesn't look like the return value is used, so would it be better to use `.any?` (same performance because it uses detect logic under the covers)